src/middlewared/middlewared/alert/source/multipaths.py

```diff
@@ -1,16 +1,19 @@
-from middlewared.alert.base import Alert, AlertLevel, AlertSource
+from middlewared.alert.base import AlertClass, AlertCategory, AlertLevel, Alert, AlertSource
 
 
-class MultipathsAlertSource(AlertSource):
+class MultipathIsNotOptimalAlertClass(AlertClass):
+    category = AlertCategory.HARDWARE
     level = AlertLevel.CRITICAL
-    title = "Multipath is not optimal"
+    title = "Multipath Is Not Optimal"
+    text = "Multipath %s is not optimal."
```

Comment:
This scares the user but doesn't give them any information to solve the problem.  We should at least suggest the most likely cause of this error.

Reply:
This usually happens when one of the links is degraded. I believe we could says something alone the lines of "check your disks cabling."